docs/csharp/language-reference/compiler-messages/snippets/UsingDirectives/MyClass.cs

```diff
@@ -1,4 +1,5 @@
 ﻿// <UsingExample>
+using System;
```

Comment:
@BillWagner The [example with the compiler error](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/using-directive-errors#using-directive) uses `using System.Text.Json;`. I suggest to either use that here too, or also change the other example.

Reply:
#35612